lametro/models/cms.py

```diff
@@ -57,3 +59,75 @@ def content(self):
             + f"<span class='button button-small alert-{self.type}'>{self.get_type_display()}</span>"
             + "</div>"
         )
+
+
+class CheckboxSelectMultipleList(forms.CheckboxSelectMultiple):
+    def format_value(self, value):
+        if isinstance(value, str):
+            return value.split(",")
```

Comment:
This is helping to actually display our arrayfield choices in the create/edit views